meson.build

```diff
@@ -57,6 +58,16 @@ if ['libicu', 'auto'].contains(enable_runtime)
   endif
 endif
 
+if ['libicu-win', 'auto'].contains(enable_runtime)
+  libicu_win_dep = cc.find_library('icu', required: false)
```

Comment:
Please also skip this for "auto" when the host machine isn't Windows, as it can't be found anyway. Result: configuring on Linux or macOS or BSD with runtime == auto is a bit faster due to running one less check.